packages/ts/form/package.json

```diff
@@ -67,24 +69,23 @@
     "access": "public"
   },
   "dependencies": {
-    "@hilla/frontend": "*",
-    "rimraf": "^5.0.1",
+    "@hilla/frontend": "~2.2",
     "validator": "^13.11.0"
   },
   "peerDependencies": {
     "lit": "^2.3.0"
   },
   "devDependencies": {
     "@esm-bundle/chai": "^4.3.4-fix.0",
-    "@types/chai": "^4.3.5",
-    "@types/chai-dom": "^1.11.0",
-    "@types/mocha": "^10.0.1",
-    "@types/sinon": "^10.0.16",
-    "@types/sinon-chai": "^3.2.9",
-    "@types/validator": "^13.11.1",
+    "@types/chai": "^4.3.6",
+    "@types/chai-dom": "^1.11.1",
+    "@types/mocha": "^10.0.2",
+    "@types/sinon": "^10.0.17",
+    "@types/sinon-chai": "^3.2.10",
+    "@types/validator": "^13.11.2",
     "chai-dom": "^1.11.0",
-    "sinon": "^15.2.0",
+    "sinon": "^16.0.0",
     "sinon-chai": "^3.7.0",
-    "typescript": "^5.1.6"
+    "typescript": "^5.2.2"
```

Comment:
Artur reverted that update since Flow is using `5.1.6` for now. So I suggest using the same TS version here (without `^` btw).

Reply:
According to https://github.com/vaadin/flow/blob/24.1/flow-server/src/main/resources/com/vaadin/flow/server/frontend/dependencies/default/package.json it's 5.0.4 for Flow 24.1 at the moment, so I used that.